api/trace.proto

```diff
@@ -0,0 +1,54 @@
+// [#protodoc-title: Tracing]
+// * Tracing :ref:`architecture overview <arch_overview_tracing>`.
```

Comment:
super nit: Unless there are multiple items I would remove the '*' / bullet point.